swarm.go

```diff
@@ -19,6 +19,13 @@ type SwarmConfig struct {
 	// `Swarm.Transports.Relay` if specified.
 	DisableRelay bool `json:",omitempty"`
 
+	// EnableRelayHop makes this node act as a public relay, relaying
+	// traffic between other nodes.
+	//
```

Comment:
This can be removed since it is impossible to actually use it, it is just here to provide migration path for existing users, we don't expect new ones.

```suggestion
```